drivers/client_access_control/extensions/grda_warehouse/hud/client_extension.rb

```diff
@@ -143,7 +143,11 @@ def enrollments_for_rollup(user:, en_scope: scope, include_confidential_names: f
             # days included in adjusted days that are not also served by a residential project
             adjusted_dates_for_similar_programs = adjusted_dates(dates: dates_served, stop_date: count_until)
             homeless_dates_for_enrollment = adjusted_dates_for_similar_programs - residential_dates(enrollments: enrollments)
-            most_recent_service = dates_served.max
+            most_recent_service = if GrdaWarehouse::Config.get(:ineligible_uses_extrapolated_days)
```

Comment:
Maybe add a comment here noting that the days served count will include days after the service date if this is set?